config/3.2/cp4waiops/templates/patch-serviceaccount.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cp4waiops-patch
+  annotations:
+    argocd.argoproj.io/sync-wave: "130"
```

Comment:
@liyanwei93 does the `sync-wave` here can guarantee the job can be finished first before creating other resources?

Reply:
No, the logic is that after the Installation CR is created, this Job will be created, which will be the last task executed on gitops. The content of the job is that during the waiops installation process, wait for the service accounts `aiops-topology-service-account` and `aiops-ir-*` to be created successfully, then patch it to fix the insufficient serviceaccount authority.